Samples/PublicSamples/RecordingBot/docs/tutorials/deploy/aks.md

```diff
@@ -0,0 +1,719 @@
+
+We can find the managed resource group in the description of the AKS cluster:
+
+```powershell
+az aks show
+    --resource-group recordingbottutorial
+    --name recordingbotcluster
+    --subscription "recordingbotsubscription" | Select-String 
+            -Pattern 'nodeResourceGroup' 
+            -SimpleMatch
+            -NoEmphasis
+```
+
+This will give us the response
+
+```json
+  "nodeResourceGroup": "MC_recordingbottutorial_recordingbotcluster_westeurope",
+```
+
+as the node resource group is the resource group where our cluster manges the resources, in our case the resource group is called `MC_recordingbottutorial_recordingbotcluster_westeurope`.
+
+### Get the public IP resource name
+
+Next we need the resource name of the public IP. To get the name we list the public IP resources in the managed resource group.
+
+```powershell
+az network public-ip list 
+    --resource-group MC_recordingbottutorial_recordingbotcluster_westeurope
+    --subscription "recordingbotsubscription"
+```
+
+The resulting list
+
+```json
+[
+  {
+    "ddosSettings": {
+      "protectionMode": "VirtualNetworkInherited"
+    },
+    "etag": "W/\"f5508f05-0e65-479a-9399-d436f02e0a66\"",
+    "id": "/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy/resourceGroups/MC_recordingbottutorial_recordingbotcluster_westeurope/providers/Microsoft.Network/publicIPAddresses/cab190bb-ec74-478e-b7f1-b36c83bfa94e",
+    "idleTimeoutInMinutes": 4,
+    "ipAddress": "108.141.184.42",
+    "ipConfiguration": {
+      "id": "/subscriptions/yyyyyyyy-yyyy-yyyy-yyyy-yyyyyyyyyyyyy/resourceGroups/MC_recordingbottutorial_recordingbotcluster_westeurope/providers/Microsoft.Network/loadBalancers/kubernetes/frontendIPConfigurations/cab190bb-ec74-478e-b7f1-b36c83bfa94e",
+      "resourceGroup": "MC_recordingbottutorial_recordingbotcluster_westeurope"
+    },
+    "ipTags": [],
+    "location": "westeurope",
+    "name": "cab190bb-ec74-478e-b7f1-b36c83bfa94e",
+    "provisioningState": "Succeeded",
+    "publicIPAddressVersion": "IPv4",
+    "publicIPAllocationMethod": "Static",
+    "resourceGroup": "MC_recordingbottutorial_recordingbotcluster_westeurope",
+    "resourceGuid": "539f439e-5c50-4bc6-a4a9-fb2d102e88f3",
+    "sku": {
+      "name": "Standard",
+      "tier": "Regional"
+    },
+    "tags": {
+      "aks-managed-cluster-name": "recordingbotcluster",
+      "aks-managed-cluster-rg": "recordingbottutorial",
+      "aks-managed-type": "aks-slb-managed-outbound-ip"
+    },
+    "type": "Microsoft.Network/publicIPAddresses",
+    "zones": [
+      "3",
+      "1",
+      "2"
+    ]
+  }
+]
+```
+
+The list only has one element as our AKS cluster only created one public IP, in our case the name of the public IP is `cab190bb-ec74-478e-b7f1-b36c83bfa94e`.
+
+### Set DNS name for public IP resource
+
+With the managed resource group name and the public IP resource name, we can now set a DNS name for the public IP resource with the command:

```

Comment:
```suggestion
With the managed resource group name and the Public IP Address resource name, we can now set a DNS name for the Public IP Address resource with the command:

```

Reply:
Why do you want to write `public IP address` in upper case?

Reply:
cause i think it was done somewhere else (in this document) and we should stay consistent
it does make sense to me, since its the "name" of a resource
but its fine in eng to use lower case

It can also be used as a means to highlight certain terms though

Reply:
Throughout this tutorial it is written lowercase everywhere now, i would choose consitency now